src/modules/hardware-wallet/src/LedgerApp.ts

```diff
@@ -0,0 +1,221 @@
+// https://github.com/radixdlt/radixdlt-ledger-app/blob/improve/change_cosmos_to_radix/docs/APDUSPEC.md
+
```

Comment:
This module is an interface for using the Radix Ledger App. 

Implements relevant methods from:
https://github.com/radixdlt/radixdlt-ledger-app/blob/master/APDUSPEC.md

Several methods are just decorated versions of sendApduMsg from HardwareWallet.ts